filter/filter.go

```diff
@@ -27,6 +31,15 @@ func (f *Filter) IsExcluded(v string) bool {
 	return ok
 }
 
+// String returns an stringification of the Filter
```

Comment:
Could be nice to have it in `-v` & `-d` output,  I haven't when running the code.

Reply:
You do: `ts=2019-07-18T09:30:15.38006452Z caller=import.go:47 func=provider.Import filters="\n\tTags:    [],\n\tInclude: [aws_iam_user],\n\tExclude: [],\n"`